incubator/k8s-node-termination-handler/templates/daemonset.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Is this copy/pasta from somewhere? I don't think we should be licensing our stuff under google....

Reply:
I removed those blocks from daemonset and rbac yamls